.github/workflows/django-step-lint-check.yml

```diff
@@ -9,9 +9,6 @@ on:
       WORKING_DIRECTORY:
         required: true
         type: string
-      PYTHON_VERSION:
```

Comment:
non veniva utilizzato (probabilmente mi era rimasto indietro), quindi visto che siamo in tempo ho fatto pulizia